patches/0005-Fix-malloc-functions.patch

```diff
@@ -0,0 +1,286 @@
+From 73ef5b1abdec1f9ada54a8b97d298b70c01c6844 Mon Sep 17 00:00:00 2001
+From: Salar Samani <soltanisamani@gmail.com>
+Date: Sun, 22 Jun 2025 07:00:51 +0200
+Subject: [PATCH 05/29] Fix malloc functions
+
```

Comment:
Does this use unikrafts internal memory allocator? Was it not easies to just use libcs malloc functions?